test/mocha.opts

```diff
@@ -0,0 +1 @@
+--require source-map-support/register
```

Comment:
We were already building sourcemaps, but we need this to make stack traces actually use them.